LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019-2020 Alexander Medvednikov
+Copyright (c) 2019-2021 Alexander Medvednikov
```

Comment:
should we have a timespan? that would need to be updated every year

Reply:
@medvednikov

Reply:
Else we would need to update it every year too and just replace `(c) 2020` with `(c) 2021`, etc.